igvc_navigation/src/action_server/action_server.h

```diff
@@ -19,6 +19,8 @@ class ActionServer
 {
 public:
   ActionServer();
+  ros::Subscriber rviz_sub;
+  ros::Subscriber waypoint_sub;
```

Comment:
Actually according to ROS style conventions members should end with a `_`, so ie. `rviz_sub_` and `waypoint_sub_`